kiwi/system/root_init.py

```diff
@@ -84,6 +85,8 @@ def create(self):
             data.sync_data(
                 options=['-a', '--ignore-existing']
             )
+            if Defaults.is_buildservice_worker():
+                copy('/.buildenv', self.root_dir)
```

Comment:
Yes I already thought about that, it is a matter of understanding what is .buildenv doing, I am not sure if .buildenv is something specific to the buildservice and its internal needs (this is what I initially thought and the main reason to limit the change only running inside the buildservice). On the other hand this file/flag is used by packages to figure out if they are being installed in some kind of chroot (or similar build/devel environment), in which case then KIWI should always make use of it.